ui/src/react-admin/modules/content-page/types/content-pages.types.ts

```diff
@@ -33,13 +34,21 @@ export type ContentOverviewTableCols =
 	| 'createdAt'
 	| 'updatedAt'
 	| 'isPublic'
+	| 'translations'
 	| 'publishedAt'
 	| 'publishAt'
 	| 'depublishAt'
 	| 'labels'
 	| 'userGroupIds'
 	| 'actions';
 
+export const NOT_TRANSLATION_PREFIX = 'NOT_';
+
+// NL, EN, NOT_NL, NOT_EN
+export type TranslationFilterValue =
+	| LanguageCode
+	| `${typeof NOT_TRANSLATION_PREFIX}${LanguageCode}`;
```

Comment:
Is this correct behavior?

![Screenshot 2024-05-10 at 15 12 32](https://github.com/viaacode/react-admin-core-module/assets/90180417/92501f0d-7ad6-4f94-8682-fb7ee0b1dce9)

Reply:
not sure what the screenshot shows
but these are string literals in typescript
It allows typescript to check the type of concatinated strings and enums
![image](https://github.com/viaacode/react-admin-core-module/assets/1710840/51e1b433-ef40-410c-aec1-e57519a8f5ed)


more info: https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html

Reply:
TIL, don't like it though